07-datastructure1/palindrome-answer.py

```diff
@@ -0,0 +1,24 @@
+import copy
+
+
+def palindrome_check(ls):
+    """
+    Check if `ls` is a palindrome or not. Return `True` if ls is same when reversed,
+    otherwise return `False`.
+
+    >>> palindrome_check([0, 1, 2])
+    False
+    >>> palindrome_check([0, 1, 0])
+    True
+    >>> palindrome_check([1, 2, 3, 2, 1])
+    True
+    """
+    reversed_list = copy.copy(ls)
+    reversed_list.reverse()
```

Comment:
Can you also add another answer which does not use internal method?

Reply:
Sure. I added an alternative answer as a comment.